### PR TITLE
feat: add ability to change the link on vmui

### PR DESCRIPTION
**Pull request by Loori-R**
Added the ability to change the `vmui link` in the datasource settings.

image
- #61

### CODE REVIEW

**Review by dmitryk-dk — DISMISSED** [review]
LGTM!

**Review by hagen1778 — DISMISSED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## tip
 
 * FEATURE: add support MetricsQL to query builder. See [this issue](https://github.com/VictoriaMetrics/grafana-datasource/issues/66).
+* FEATURE: Add the ability to change the link on [vmui](https://docs.victoriametrics.com/#vmui). See [this issue](https://github.com/VictoriaMetrics/grafana-datasource/issues/61).
```
Comment (hagen1778) [inline CHANGELOG.md:6]:
```suggestion
* FEATURE: Add the ability to change the link for [Run in VMUI](https://docs.victoriametrics.com/#vmui) button. See [this issue](https://github.com/VictoriaMetrics/grafana-datasource/issues/61).
```

**Review by hagen1778 — APPROVED** [review]

**Comment by hagen1778** [comment]
@dmitryk-dk would you care to take a look?

**Pull request merged**